presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -64,6 +64,10 @@ class TaskManager {
           std::unordered_map<std::string, std::string>>&&
           connectorConfigStrings);
 
+  void processResultRequests(
```

Comment:
Would you document this method to explain how it handles the case when task has finished (failed) before this method was invoked?

Looks like this is a public method. I'd expect it to be private, but I assume it is used in the test. If that's the case, let's comment to clarify that this method is made public only for access from the test and should not be used by any production code.